src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -692,17 +691,13 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
         });
         const caseIDResult = await this.props.downloadRawDataByFields({ fields: [caseField] });
         if (caseIDResult) {
-          let caseIDList = caseIDResult.map((i) => i[caseField]);
-          caseIDList = _.uniq(caseIDList);
           const fileType = this.props.guppyConfig.manifestMapping.resourceIndexType;
           if (!fileType) {
             throw Error('guppyConfig.manifestMapping.resourceIndexType is not defined');
           }
-          const countResult = await this.props.getTotalCountsByTypeAndFilter(fileType, {
-            [caseFieldInFileIndex]: {
-              selectedValues: caseIDList,
-            },
-          });
+          const countResult = await this.props.getTotalCountsByTypeAndFilter(fileType,
+            this.props.filter,
```

Comment:
I think this is a breaking change. This requires the fields that has been specified in the explorer tab of a certain index must have all those fields injected into its corresponding `File` index, otherwsie this query will fail

This works for MIDRC staging because MIDRC staging is handling its ETL mapping in a special way by injecting all these fields from `Case` index into the `File` index https://github.com/uc-cdis/cdis-manifest/blob/master/staging.midrc.org/etlMapping.yaml#L161-L183

Since this is a breaking change, it must be specified in the PR comment. And since this will affects all the non-file type data explorer tabs, I think this needs to get communicated to other project teams

Reply:
This PR was not ready to land. It was definitely a work in progress and I did not have time to look into how the changes affects data-portal. Based on @mfshao comments it seems it does and that needs to be mitigated before it can be merged.

Reply:
I have made changes that should isolate the change to only commons enabling the features.